backend/telemetry_core/src/aggregator/inner_loop.rs

```diff
@@ -100,6 +101,30 @@ pub enum FromFeedWebsocket {
     Disconnected,
 }
 
+/// A set of metrics returned when we ask for metrics
+#[derive(Clone, Debug, Default)]
+pub struct Metrics {
+    /// When in unix MS from epoch were these metrics obtained
+    pub timestamp_unix_ms: u64,
+    /// How many chains are feeds currently subscribed to.
+    pub chains_subscribed_to: usize,
+    /// How many feeds are currently subscribed to something.
+    pub subscribed_feeds: usize,
+    /// How many feeds have asked for finality information, too.
+    pub subscribed_finality_feeds: usize,
+    /// How many messages are currently queued up in internal channels
+    /// waiting to be sent out to feeds.
+    pub total_messages_to_feeds: usize,
+    /// How many messages are queued waiting to be handled by this aggregator.
+    pub total_messages_to_aggregator: usize,
+    /// How many nodes are currently known about by this aggregator.
+    pub connected_nodes: usize,
+    /// How many feeds are currently connected to this aggregator.
+    pub connected_feeds: usize,
```

Comment:
dq: what is the difference between connected feeds and subscribed feeds?

Reply:
As soon as a connection to `/feed` is established, the aggregator can see that as a connected feed. Only when the feed sends a message like "subscribe:Polkadot" does it also count as having subscribed to something!

Reply:
So one connection could subscribe to multiple chains?

Reply:
Nope; if a connection calls "susbcribe:ChainName" it'll unsubscribe them from any previous chain they were subscribed to, so it's just one at a time.

Reply:
Got it.